Formula/p/prjtrellis.rb

```diff
@@ -0,0 +1,48 @@
+class Prjtrellis < Formula
+  desc "Documenting the Lattice ECP5 bit-stream format"
+  homepage "https://github.com/SymbiFlow/prjtrellis"
+  url "https://github.com/YosysHQ/prjtrellis/archive/refs/tags/1.4.tar.gz"
+  sha256 "46fe9d98676953e0cccf1d6332755d217a0861e420f1a12dabfda74d81ccc147"
+  license "MIT"
+  head "https://github.com/YosysHQ/prjtrellis.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "python@3.12"
+
+  resource "prjtrellis-db" do
+    url "https://github.com/YosysHQ/prjtrellis/releases/download/1.4/prjtrellis-db-1.4.zip"
+    sha256 "4f8a8a5344f85c628fb3ba3862476058c80bcb8ffb3604c5cca84fede11ff9f0"
+  end
```

Comment:
Since there is a `head`, is this only needed on `stable`?

Reply:
yeah, I should remove the head build, as it would need `git submodule init`